docs/versioned_docs/version-6.3/api/methods/init-connection.md

```
@@ -0,0 +1,115 @@
+      FlutterInappPurchase.instance.endConnection();
+    }
+    super.dispose();
+  }
+}
+```
+
+## Related Methods
+
+- `endConnection()` - Closes the connection to the billing service
+- [`getProducts()`](./get-products.md) - Fetches available products (requires an initialized connection)
+- [`requestPurchase()`](./request-purchase.md) - Initiates a purchase (requires an initialized connection)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)